sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -276,6 +279,13 @@ def process(self, element):
     with self.assertRaisesRegex(ValueError, r'window.*None.*add_timestamps2'):
       pipeline.run()
 
+class AddTimestamp(beam.DoFn):
```

Comment:
As these are used only locally, define then locally (i.e. in the test method).

Reply:
Done.